### ISSUE TITLE
semicolon erased after match's braces, php8.1

### ISSUE THREAD

**Issue by jspourre** [squizlabs/PHP_CodeSniffer#3623]
**Describe the bug**
When using the match structure from php8.1 and making a phpcbf, it erase the semicolon after { } from a match

**Code sample**
```php
  match {
   con1 => ... ,
   con2 => ... ,
 };
```

**To reproduce**
1. Create a file called `test.php` using a match (https://www.php.net/manual/fr/control-structures.match.php)
2. Run `phpcs test.php ...`
```
ERROR   | [x] Useless semicolon.
```
3. when using `phpcbf test.php` go to the file and see that the `;` from the match is deleted

**Expected behavior**
expected to not erase de semicolon

**Versions (please complete the following information):**
 - OS: [ubuntu 21]
 - PHP: [8.1]
 - PHPCS: [3.7.1]
 - Standard: [PSR2]
- Symfony 5.4.6

**Comment by jrfnl** [comment]
@jspourre I cannot reproduce this issue at all with (a syntax valid) variation of the given code sample and the `PSR2` ruleset.

I also checked and PHPCS itself does not contain any sniff throwing that error message.

I suspect the error is coming from the SlevomatCodingStandard - you can check by running PHPCS with the `-s` option to see the error code.

In future, please report any issues with a code sample with which something can be reproduced. Please don't waste people's time by asking them to "make something up yourself".

**Comment by jspourre** [comment]
Sorry for the lack of information, first issue i write. 
I tryied to do a single php file with a match structure and found no bug too. 
I went back to my project and try it again and get the problem. I  saw that i use doctrine/coding-standard too in my project so i made and upgrade of it from ^8.2 to ^9.0 and that solved the problem. 
Sorry for the wast of time.

**Comment by gsherwood** [comment]
@jspourre Thanks for reporting back that you sorted it out. Glad it's working now.
